shared/common-adapters/safe-area-view.desktop.js

```diff
@@ -1,3 +1,10 @@
 // @flow
-// Just act like a box
-export {Box as default, Box as SafeAreaViewTop} from './box'
+import * as React from 'react'
+import type {SafeAreaViewTopBottomProps} from './safe-area-view'
+
+const RenderChildren = (props: SafeAreaViewTopBottomProps) => (
+  <React.Fragment>{props.children}</React.Fragment>
+)
+
+// Do nothing
+export {RenderChildren as default, RenderChildren as SafeAreaViewTop, RenderChildren as SafeAreaViewTopBottom}
```

Comment:
cc @chrisnojima I changed how this works. I think it only affects the provision code page, I checked it out and it looks good.